src/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueIssetToObjectHasAttributeRector.php

```diff
@@ -15,9 +15,11 @@
 /**
  * Before:
  * - $this->assertTrue(isset($anything->foo));
+ * - $this->assertFalse(isset($anything->foo));
  *
  * After:
  * - $this->assertObjectHasAttribute('foo', $anything);
+ * - $this->assertObjectNotHasAttribute('foo', $anything);
```

Comment:
Nice, I like this one!

Reply:
I keep this one, the other two I couldn't figure out better comments